src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -28,12 +32,81 @@ const options: Partial<Options> = {
     placement: 'bottom-start',
 };
 
-const PopupList = ({ reference, ...rest }: Props): JSX.Element => (
-    <PopupBase className="ba-PopupList" options={options} reference={reference} {...rest}>
-        <div className="ba-PopupList-prompt ba-PopupList-item">
-            <FormattedMessage {...messages.popupListPrompt} />
+const PopupList = ({ children, onSelect, reference, selector, ...rest }: Props): JSX.Element => {
+    const [activeItemIndex, setActiveItemIndex] = React.useState<number>(-1);
+
+    const handleKeyDown = (event: KeyboardEvent): void => {
+        const childrenCount = React.Children.count(children);
+
+        const stopDefaultEvent = (e: KeyboardEvent): void => {
+            e.preventDefault();
+            e.stopPropagation();
+        };
+
+        switch (event.key) {
+            case 'ArrowDown':
+                if (childrenCount) {
+                    stopDefaultEvent(event);
+                }
+                setActiveItemIndex(activeItemIndex === childrenCount - 1 ? -1 : activeItemIndex + 1);
```

Comment:
if `childrenCount` is falsy, should `setActiveItemIndex` still be called?

Reply:
Also, if `activeItemIndex === childrenCount - 1`, does that mean it would be at the end of the list? In which case, should `ArrowDown` then go back to index 0?